src/main/webapp/src/app/my_gardens_page/my-gardens.component.ts

```diff
@@ -0,0 +1,26 @@
+import {Component, OnInit} from '@angular/core';
+import {OwlOptions} from 'ngx-owl-carousel-o';
+
+@Component({
+  selector: 'MyGardens',
+  templateUrl: './my-gardens.component.html',
+  styleUrls: ['./my-gardens.component.css'],
+})
+export class MyGardensComponent implements OnInit {
+  /* Option to display a certian number of slides based off of browser window size along with
```

Comment:
In addition to the spelling nit @samouri pointed out, another nit that applies to both these PRs is the comment delimiting.

Use
```javascript
/**
 * Line 1 of the comment
 * Line 2 of the comment
 * Line 3 of the comment
 */
```

Reply:
Done